source/reference/minio-cli/minio-mc-admin/mc-admin-speedtest.rst

```diff
@@ -0,0 +1,64 @@
+======================
+``mc admin speedtest``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin speedtest
+
+Description
+-----------
+
+.. start-mc-admin-speedtest-desc
+
+The :mc-cmd:`mc admin speedtest` command tests throughputs per host with PUT and GET operations.
+
+:mc-cmd:`mc admin speedtest` is not supported on standalone MinIO deployments
+or MinIO Gateway deployments.
+
```

Comment:
@Butzlabben this looks great! One minor request. We're (well, I) am not doing this very consistently, but I'd like to try including the version number in which a feature was added. It helps guide users who are confused as to why they don't see a given <thing>. You can do this like:

```suggestion

.. versionadded::

   :mc-release:`RELEASE.2021-09-02T09-21-27Z`

```
```

Reply:
We also need to add in this section that the speedtest command only works for MinIO servers running:

```
:minio-release:`RELEASE.2021-07-30T00-02-00Z` or later.
```